priv/cabbage/apps/itest/test/itest/transactions_test.exs

```diff
@@ -73,46 +73,6 @@ defmodule TransactionsTests do
     {:ok, state}
   end
 
-  defthen ~r/^they should have "(?<amount>[^"]+)" ETH on the child chain$/,
```

Comment:
see line: 143. It is the same, and this is actually not used.

Reply:
good catch, I did this! :D